twitter_timeline/utils.py

```diff
@@ -33,6 +33,13 @@ def auth_only(f):
     @wraps(f)
     def decorated_function(*args, **kwargs):
         # implement your logic here
+        if 'Authorization' not in request.headers:
+            return Response('No Authorization field in header',401)
+        client_access_token = request.headers['Authorization']
+        cursor = g.db.auth.find_one({'access_token': client_access_token})
+        if not cursor:
+            return Response('Access token does not match', 401)
+        kwargs['user_id'] = cursor['user_id']
```

Comment:
Interesting!  I just included this as a positional argument and didn't think about using at a keyword argument.  I think using a keyword argument aligns a little better with "explicit is better than implicit."